Easydict/App/Localizable.xcstrings

```diff
@@ -4231,7 +3729,7 @@
         "zh-Hans" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "静默截图 OCR"
+            "value" : "静默截图OCR"
```

Comment:
We already have this string in SwiftUI, so it's ok remove these old strings.

<img width="1059" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/b13d78b8-2070-44e8-8b2f-ec58fc5e4888">